modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/InfoPanel.es.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from "react";
+
+const InfoPanel = ({ imageData }) => {
+
+	const TPL_INFO_TAB_BODY =
+			'<div class="{className} fade in tab-pane" id="{tabId}">{content}</div>';
+
+	const TPL_INFO_TAB_BODY_CONTENT =
```

Comment:
Are `TPL_INFO_TAB_BODY` and `TPL_INFO_TAB_BODY_CONTENT` unused? Is `imageData` unused? Shoulnd't this be called `InfoPanelTitle` instead?